### ISSUE TITLE
Make VPN

### ISSUE THREAD

**Issue by Acceptto** [Acceptto/BrowserBoxAction#1]
# Set Up a VPN

Before starting, [fork](../fork) or [generate](../generate) this repo to your account (not for orgs).

Then ensure that:

1. [Issues](../settings#issue-feature) are switched on, and 
2. [Actions](actions) are enabled.

To begin the action to create your BrowserBox Tor Hidden Service, click **Submit New Issue**. 

Keep an eye on the comments for next steps.

*Note: This VPN activates only for the repo owner when opening or reopening the issue. It's designed to conserve your GitHub Actions minutes. My username has access for support purposes. Adjust `userjob.yaml` if you wish to change this.*

Once you click **Submit New Issue**, follow-up instructions will appear in the comments.

---

*Powered by [DOSYAGO](https://dosyago.com)*
